src/olympia/migrations/971-add-text-reviews-count-to-addon.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE addons
+    ADD COLUMN `textreviews` int(11) DEFAULT 0;
```

Comment:
Need to match the new field name

Reply:
Also, need to be migration 972

Reply:
I wasn't going to change the db column, heh.  But sure, can do.